includes/modules/additional_images.php

```diff
@@ -94,13 +96,19 @@
 if ($num_images > 0) {
     $row = 0;
     $col = 0;
-    if ($num_images < IMAGES_AUTO_ADDED || IMAGES_AUTO_ADDED == 0 ) {
-        $col_width = floor(100/$num_images);
-    } else {
-        $col_width = floor(100/IMAGES_AUTO_ADDED);
+    echo '
+<style>
```

Comment:
Embedded CSS?  I realize that the html_header.php file(s) don't recognize /css/*.php files but this sounds like a recipe for failure.

Reply:
Yes, this was intentional, so that the defined config can be used in the template.
Do you have a better way of dynamically outputting an admin config setting to the CSS?

Reply:
Okay. I can accept that.  In this scenario, the generated dynamic inline CSS isn't context-specific, so doing it via `html_header` is fine.